change/@fluentui-react-charting-effaa597-2236-4e8c-b8f3-91a2422c9f44.json

```diff
@@ -0,0 +1,7 @@
+{
```

Comment:
## [🕵 fluentuiv9 Open the Visual Regressions report to inspect the 7 screenshots]( https://staticpagerelease.z16.web.core.windows.net/azurestaticapps/vr-fluent/index.html?organization=uifabric&project=fabricpublic&prId=27415&commitId=0ddde0ffd6a3c3ffea3220a0958d976c201f5cf6&env=prod)

✅ There was 0 screenshots added, 0 screenshots removed, 1924 screenshots unchanged, 0 screenshots with different dimensions and 7 screenshots with visible difference.


  <details><summary><b>unknown</b> 7 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|SpinButton Converged.Appearance- filled-darker - Dark Mode.mouseDownIncrement.chromium.png|1831822| Changed|
|SpinButton Converged.Appearance- filled-darker - High Contrast.mouseDownIncrement.chromium.png|1832370| Changed|
|SpinButton Converged.Appearance- filled-darker.mouseDownDecrement.chromium.png|1827814| Changed|
|SpinButton Converged.Appearance- filled-lighter.mouseDownIncrement.chromium.png|1821432| Changed|
|SpinButton Converged.Appearance- underline - Dark Mode.mouseDownDecrement.chromium.png|1831485| Changed|
|SpinButton Converged.Appearance- underline - High Contrast.mouseDownDecrement.chromium.png|1832400| Changed|
|SpinButton Converged.Appearance- underline - RTL.mouseDownDecrement.chromium.png|1827830| Changed|

</details>


<div id="vrtCommentfluentuiv9"/>